windows/src/desktop/help/basic/config/keyboards.md

```diff
@@ -60,23 +86,32 @@ Keyman Configuration:
 
 1.  Find a keyboard in the keyboard list.
 
-2.  Click on the down arrow
-    (![](../../desktop_images/keyboards-downarrow.png)) beside the keyboard
-    name. The keyboard information window will expand.
-
-    ![](../../desktop_images/keyboards-hotkey.png)
-
-3.  In the information window you can view keyboard details, including:
-
+2.  Select the keyboard name from the keyboard list to expand options associated with it. 
+    The keyboard information window will expand.
+    (![](../../desktop_images/tab-layout-detail1.png)) 
+    The initial drop down has the following information if available: 
+    
+      -   Languages associated with the keyboard 
+      -   Keyboard package name
+      -   Keyboard package version number
+      -   Copyright details
+        
+
+  
+3. Click on the down arrow
+    (![](../../desktop_images/keyboards-downarrow.png)) at the end of the list will reveal
```

Comment:
```suggestion
    ![](../../desktop_images/keyboards-downarrow.png) at the end of the list will reveal
```

Depends if you're wanting parens around the down arrow image.

Reply:
Yeah the exiting md that had the "arrow" and "?" had the extra rendered parentheses. Then all references to the keyman icon have no parentheses and just a comma.  I think will remove the parentheses as the icon does not get rendered in the centre so it looks strange.